Example/Shared/FIRComponentTestUtilities.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+// Include any interop interfaces if they're available for import.
```

Comment:
I'm not a fan of this kind of magic. It makes it very hard to actually depend upon a value being in scope.

Better to either import this as a part of this contract or force callers that care about auth interop to import it for themselves.

I generally favor IWYU. It's explicit, obvious, and tool enforceable.

(Note that this feedback does not necessarily apply to our *public* interface, where we may want to make it easier for people to get started. Internally though, we should be precise.)

Reply:
SGTM! Removed.